### PR TITLE
Feature/keriat zkf 262 zksyncio

### PR DESCRIPTION
**Pull request by keriat**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,5 @@
 SENTRY_DSN=https://1f18c5c8e8fb4f2491be8d9daac67145@o496053.ingest.sentry.io/5635442
 SITE_TITLE="zkSync — Rely on math, not validators"
-SITE_DESCRIPTION="zkSync is a ZK rollup that solves Ethereum’s scalability and UX — without security compromises"
+SITE_DESCRIPTION="zkSync is a ZK rollup solving Ethereum scalability with zero security compromises"
```
Comment (gluk64) [inline .env.example]: Точку нужно в конце.

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+SITE_KEYWORDS="zkSync, Matter Labs, rollup, ZK rollup, zero confirmation, Ethereum, crypto, blockchain, permissionless, L2, secure payments, scalable crypto payments"
```
Comment (gluk64) [inline .env.example]: "zero confirmation" => "ZKP, zero-knowledge proofs"

File: src/components/ZHero.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
     </h1>
     <p class="grayText aboutZkSync subheaderText" data-aos="fade-up" data-aos-delay="100" data-aos-duration="2000">
       <emphasis :show-logo="true"/>
-      solves Ethereum scalability<br>with zero security compromises.
+      solves Ethereum scalability<br>&nbsp;with zero security compromises
```
Comment (gluk64) [inline src/components/ZHero.vue:8]: Почему без точки?

**Review by gluk64 — CHANGES_REQUESTED** [review]

File: src/components/ZHero.vue (hunk 3 above)
Comment (keriat) [reply src/components/ZHero.vue:8]: Так единообразно. Все subtitles ниже без точки.

**Pull request merged**